include/CoreGraphics/CGContext.h

```diff
@@ -193,11 +193,11 @@ COREGRAPHICS_EXPORT void CGContextStrokeEllipseInRect(CGContextRef c, CGRect rec
 COREGRAPHICS_EXPORT void CGContextStrokeLineSegments(CGContextRef c, const CGPoint* points, size_t count);
 COREGRAPHICS_EXPORT bool CGContextIsPathEmpty(CGContextRef c);
 
-COREGRAPHICS_EXPORT CGPoint CGContextGetPathCurrentPoint(CGContextRef c) STUB_METHOD;
```

Comment:
Does this mean these APIs don't have tests? The compiler would have warned the usage of these if they were marked stub.